File: java/src/main/java/org/example/collection/QuickCollection.java
package org.example.collection;

import io.milvus.v2.common.DataType;
import io.milvus.v2.client.ConnectConfig;
import io.milvus.v2.client.MilvusClientV2;
import io.milvus.v2.service.collection.request.GetLoadStateReq;
import io.milvus.v2.service.collection.request.CreateCollectionReq;


public class QuickCollection {
    private static final MilvusClientV2 client;
    static {
        String CLUSTER_ENDPOINT = "http://localhost:19530";
        String TOKEN = "root:Milvus";
        client = new MilvusClientV2(ConnectConfig.builder()
                .uri(CLUSTER_ENDPOINT)
                .token(TOKEN)
                .build());
    }

    private static void simpleCreate() {
        // 2. Create a collection in quick setup mode
        CreateCollectionReq quickSetupReq = CreateCollectionReq.builder()
                .collectionName("quick_setup")
                .dimension(5)
                .build();

        client.createCollection(quickSetupReq);

        GetLoadStateReq quickSetupLoadStateReq = GetLoadStateReq.builder()
                .collectionName("quick_setup")
                .build();

        Boolean res = client.getLoadState(quickSetupLoadStateReq);
        System.out.println(res);
    }

    private static void quickCreate() {
        // 2. Create a collection in quick setup mode
        CreateCollectionReq customQuickSetupReq = CreateCollectionReq.builder()
                .collectionName("custom_quick_setup")
                .dimension(5)
                .primaryFieldName("my_id")
                .idType(DataType.VarChar)
                .maxLength(512)
                .vectorFieldName("my_vector")
                .metricType("L2")
                .autoID(true)
                .build();

        client.createCollection(customQuickSetupReq);

        GetLoadStateReq customQuickSetupLoadStateReq = GetLoadStateReq.builder()
                .collectionName("custom_quick_setup")
                .build();

        Boolean res = client.getLoadState(customQuickSetupLoadStateReq);
        System.out.println(res);
    }

    public static void main(String[] args) {
        simpleCreate();
        quickCreate();
    }
}

File: python/collection/quick_collection.py
from pymilvus import MilvusClient, DataType

CLUSTER_ENDPOINT = "http://localhost:19530"
TOKEN = "root:Milvus"

# 1. Set up a Milvus client
client = MilvusClient(
    uri=CLUSTER_ENDPOINT,
    token=TOKEN
)

def create_simple():
    # 2. Create a collection in quick setup mode
    client.create_collection(
        collection_name="quick_setup",
        dimension=5
    )

    res = client.get_load_state(
        collection_name="quick_setup"
    )

    print(res)

def create_custom():
    # 2. Create a collection in quick setup mode
    client.create_collection(
        collection_name="custom_quick_setup",
        dimension=5,
        primary_field_name="my_id",
        id_type="string",
        vector_field_name="my_vector",
        metric_type="L2",
        auto_id=True,
        max_length=512
    )

    res = client.get_load_state(
        collection_name="custom_quick_setup"
    )

    print(res)

if __name__ == "__main__":
    create_simple()
    create_custom()